openedx/core/lib/block_structure/cache.py

```diff
@@ -41,12 +46,13 @@ def add(self, block_structure):
         data_to_cache = (
             block_structure._block_relations,
             block_structure._transformer_data,
-            block_structure._block_data_map
+            block_structure._block_data_map,
         )
         zp_data_to_cache = zpickle(data_to_cache)
         self._cache.set(
             self._encode_root_cache_key(block_structure.root_block_usage_key),
-            zp_data_to_cache
+            zp_data_to_cache,
+            timeout=self.cache_timeout,
```

Comment:
The latter. I haven't really followed Block Transformers work very closely, so this might just be my own ignorance speaking, but I can't think of a scenario where we'd want to override this setting in this way. Can't we just keep it as an implementation detail?